options/options.go

```diff
@@ -203,6 +203,10 @@ type TerragruntOptions struct {
 	// Unix-style glob of directories to include when running *-all commands
 	IncludeDirs []string
 
+	// If set to true, exclude all directories by default when running *-all commands
+	// Is set automatically if IncludeDirs is set
+	ExcludeByDefault bool
```

Comment:
Usually, when a field is added to `TerragruntOptions`, it should be also updated `Clone()` function to copy the field